osu.Game/Graphics/UserInterface/ContextMenuItem.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input;
+using osu.Game.Graphics.Sprites;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class ContextMenuItem : ClickableContainer
+    {
+        private const int height = 25;
+        private const int width = 170;

```

Comment:
Context menus should scale to the width of the largest item, not a static width.